extensions/wikia/ArticleVideo/ArticleVideo.hooks.php

```diff
@@ -20,20 +20,13 @@ public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $ou
 		if ( isset( $wg->articleVideoFeaturedVideos[$title]['videoId'] ) ) {
 			$vars['wgArticleVideoData'] = [
 				'videoId' => $wg->articleVideoFeaturedVideos[$title]['videoId'],
+				'playerParams' => [
+					'ooyalaPCode' => $wg->ooyalaApiConfig['pcode'],
+					'ooyalaPlayerBrandingId' => $wg->ooyalaApiConfig['playerBrandingId'],
```

Comment:
I don't think it fits `wgArticleVideoData` as it's not really _a video data_ but rather _the player config_.